src/helpers/app.ts

```diff
@@ -0,0 +1,16 @@
+const DEFAULT_SUBDOMAIN = 'app'
```

Comment:
Can you export this constant too, and maybe rename it to `DEFAULT_DATADOG_SUBDOMAIN`?